apps/client-ui/scripts/generate-amplify-config.sh

```
@@ -0,0 +1,371 @@
+                shift 2
+                ;;
+            --pr-number)
+                PR_NUMBER="$2"
+                shift 2
+                ;;
+            --output-file)
+                OUTPUT_FILE="$2"
+                shift 2
+                ;;
+            --validate-only)
+                validate_only="true"
```

Shellcheck (0.10.0)

[warning] validate_only appears unused. Verify use (or export if used externally).

(SC2034)